docs/source/changelog.rst

```diff
@@ -1,17 +1,12 @@
 Changelog
 =========
 
-.. X.Y.0 / 2021-MM-DD
-.. -------------------
-..
-.. New Features
-.. ++++++++++++
-..
-.. Enhancements
-.. ++++++++++++
-..
-.. Bug Fixes
-.. +++++++++
+X.Y.0 / 2021-MM-DD
+-------------------
+
+New Features
+++++++++++++
+- (:pr:`268`) Add provisional models that store the inputs to and outputs of a torsion drive procedure.
```

Comment:
```suggestion
.. X.Y.0 / 2021-MM-DD
.. -------------------
..
.. New Features
.. ++++++++++++
..
.. Enhancements
.. ++++++++++++
..
.. Bug Fixes
.. +++++++++


0.22.0 / 2021-MM-DD
-------------------

New Features
++++++++++++
- (:pr:`268`) Add provisional models that store the inputs to and outputs of a torsion drive procedure. @SimonBoothroyd
```

Reply:
Oops sorry I didn't mean to delete the template - great catch!